SHSearchBar/Classes/SHSearchBar.swift

```diff
@@ -22,11 +22,14 @@ public class SHSearchBar: UIView, SHSearchBarDelegate {
     var bgToParentConstraint: NSLayoutConstraint!
 
     /// The background image view which is responsible for displaying the rounded corners.
-    let backgroundView: UIImageView = UIImageView()
+    public let backgroundView: UIImageView = UIImageView()
 
     /// The cancel button under the right side of the searhcbar.
-    let cancelButton: UIButton = UIButton(type: .custom)
+    public let cancelButton: UIButton = UIButton(type: .custom)
```

Comment:
It is not intended to make this view public. Have a look at the SHSearchBarConfig regarding customization options.

Why do you need that?